### ISSUE TITLE
autograd 관련 질문 (서유빈)

### ISSUE THREAD

**Issue by youbin4510** [codingalzi/dlp2#9]
<img width="958" height="1022" alt="Image" src="https://github.com/user-attachments/assets/2232bcac-4a63-49c5-b0a1-a466be4a895c" />

서유빈입니다.

위에 코드는 3장 강의자료에 없는 부분이라서 찾아보니 requires_grad=True는 연산을 추적해서 미분을 하고, no_grad()나 detach()는 연산을 추적하지 않는다는 걸 알았습니다.
autograd가 자동으로 미분을 해주는 것이니 경사하강법에서 사용되는 것 같은데 실제로는 어떻게 활용되는지 알고 싶습니다.

**Comment by liganega** [comment]
아래 링크 읽어 보삼.

https://colab.research.google.com/github/codingalzi/dlp2/blob/master/notebooks/NB-PyTorch_Autograd_tutorial.ipynb

**Comment by liganega** [comment]
온라인강의 3주 1차시 동영상도 시청하고.
